govcd/vm.go

```diff
@@ -1501,3 +1501,38 @@ func (client *Client) QueryVmList(filter types.VmQueryFilter) ([]*types.QueryRes
 	}
 	return vmList, nil
 }
+
+// UpdateVmCapabilities updates VM Capabilities and returns refreshed VM or error.
+func (vm *VM) UpdateVmCapabilities(cpuHot, memoryHot bool) (*VM, error) {
```

Comment:
Are these (cpu and memory hot plug)  all capabilities a VM can have set? Asking because I would think of a more precise function name instead of generalized `UpdateVmCapabilities`

Reply:
These two settings are all Capabilities support currently. I named by XML structure it covers.
I don't buy it with naming `plug` we don't have in UI(`Virtual CPU hot add`) or in API it.
My suggestions for another name would be:
`UpdateVmHotCapabilities`
`UpdateVmCpuAndMemoryHotAdd`

Reply:
I vote for `UpdateVmCpuAndMemoryHotAdd` - @dataclouder, @lvirbalas

Reply:
Sounds good!

Reply:
Sounds good!

Reply:
renamed